app.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+(function(){
+  angular.module('toDoApp', []).controller('list_controller', ListController);
+  function ListController(){
+    var toggleStatus = 0;
+    var vm = this;
+    vm.tasks = tasks;
+    vm.taskAdder = function(){
+      var taskBox = document.getElementById('task-field').value;
+      var taskNotes = document.getElementById('add-notes-field').value;
+      if(taskBox !== '' || taskNotes !== '') {
+        vm.tasks.push({
+          name: taskBox,
+          notes: taskNotes
+        });
+      }
+      document.getElementById('task-field').value = '';
+      document.getElementById('add-notes-field').value = '';
+    };
+    vm.deleteTask = function(task){
+      console.log('hey');
+      var index = vm.tasks.indexOf(task);
+      vm.tasks.splice(index, 1);
+    };
+    vm.editTask = function(task){
+      var index = vm.tasks.indexOf(task)
+      if(toggleStatus == 0){
```

Comment:
Could you think of another way to solve this without using `toggleStatus` ? Also, you I would recommend you call the `editTask` method on button itself or just the input to just simply.